src/clr-angular/data/datagrid/datagrid-column-toggle-button.ts

```diff
@@ -10,35 +10,19 @@ import { ColumnToggleButtons, ColumnToggleButtonsService } from './providers/col
 @Component({
   selector: 'clr-dg-column-toggle-button',
   template: `
-        <button
+        <button class="btn btn-sm btn-link p6 text-uppercase"
```

Comment:
Why did we add `.p6` and `.text-uppercase` here?

Aren't those implicit in the styling of a `.btn-sm`?

Reply:
I copied over the class we had for the "select all" button. Let me evaluate if those classes (`.p6` and `.text-uppercase` are really needed.